### PR TITLE
bitopro: add cancelOrders api

### PR DESCRIPTION
**Pull request by sc0Vu**
Add cancelOrders in bitopro

https://github.com/bitoex/bitopro-offical-api-docs/blob/master/v3-1/rest-1/auth/cancel-batch.md

### CODE REVIEW

File: js/bitopro.js [hunk 1]
```diff
@@ -950,6 +951,29 @@ module.exports = class bitopro extends Exchange {
...
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrders() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const id = market['id'].toUpperCase ();
+        const request = {};
+        request[id] = ids;
+        const response = await this.privatePutOrders (this.extend (request, params));
```
Comment (kroitor) [inline js/bitopro.js:963]: I think `privatePutOrders` is used for order-editing (changing the amounts or prices of the open orders), and for canceling we need to use the `delete` methods?

Comment (sc0Vu) [reply js/bitopro.js:963]: Yes, if they follow the REST api specification (which they didn't). I felt weird to use put......

Comment (kroitor) [reply js/bitopro.js:963]: What about this endpoint is it the same functionality does it work? 

- https://github.com/bitoex/bitopro-offical-api-docs/blob/master/v3-1/rest-1/auth/cancel-all.md

Comment (kroitor) [reply js/bitopro.js:963]: Actually, you're right, nevermind.

**Pull request merged**